handler/src/main/java/io/netty/handler/ssl/ReleasableOpenSslEngine.java

```diff
@@ -0,0 +1,1988 @@
+
+    private static final SSLException BEGIN_HANDSHAKE_ENGINE_CLOSED = ThrowableUtil.unknownStackTrace(
+            new SSLException("engine closed"), ReleasableOpenSslEngine.class, "beginHandshake()");
+    private static final SSLException HANDSHAKE_ENGINE_CLOSED = ThrowableUtil.unknownStackTrace(
+            new SSLException("engine closed"), ReleasableOpenSslEngine.class, "handshake()");
+    private static final SSLException RENEGOTIATION_UNSUPPORTED =  ThrowableUtil.unknownStackTrace(
+            new SSLException("renegotiation unsupported"), ReleasableOpenSslEngine.class, "beginHandshake()");
+    private static final SSLException ENCRYPTED_PACKET_OVERSIZED = ThrowableUtil.unknownStackTrace(
+            new SSLException("encrypted packet oversized"), ReleasableOpenSslEngine.class, "unwrap(...)");
+    private static final Class<?> SNI_HOSTNAME_CLASS;
+    private static final Method GET_SERVER_NAMES_METHOD;
+    private static final Method SET_SERVER_NAMES_METHOD;
+    private static final Method GET_ASCII_NAME_METHOD;
+    private static final Method GET_USE_CIPHER_SUITES_ORDER_METHOD;
+    private static final Method SET_USE_CIPHER_SUITES_ORDER_METHOD;
+    private static final AtomicIntegerFieldUpdater<ReleasableOpenSslEngine> refCntUpdater;
+    private static final ResourceLeakDetector<ReleasableOpenSslEngine> leakDetector =
+            ResourceLeakDetectorFactory.instance().newResourceLeakDetector(ReleasableOpenSslEngine.class);
+
+    static {
+        AtomicIntegerFieldUpdater<ReleasableOpenSslEngine> destroyedUpdater =
+                PlatformDependent.newAtomicIntegerFieldUpdater(ReleasableOpenSslEngine.class, "destroyed");
+        if (destroyedUpdater == null) {
+            destroyedUpdater = AtomicIntegerFieldUpdater.newUpdater(ReleasableOpenSslEngine.class, "destroyed");
+        }
+        DESTROYED_UPDATER = destroyedUpdater;
+
+        AtomicIntegerFieldUpdater<ReleasableOpenSslEngine> updater =
+                PlatformDependent.newAtomicIntegerFieldUpdater(ReleasableOpenSslEngine.class, "refCnt");
+        if (updater == null) {
+            updater = AtomicIntegerFieldUpdater.newUpdater(ReleasableOpenSslEngine.class, "refCnt");
+        }
+        refCntUpdater = updater;
+
+        Method getUseCipherSuitesOrderMethod = null;
+        Method setUseCipherSuitesOrderMethod = null;
+        Class<?> sniHostNameClass = null;
+        Method getAsciiNameMethod = null;
+        Method getServerNamesMethod = null;
+        Method setServerNamesMethod = null;
+        if (PlatformDependent.javaVersion() >= 8) {
+            try {
+                getUseCipherSuitesOrderMethod = SSLParameters.class.getDeclaredMethod("getUseCipherSuitesOrder");
+                SSLParameters parameters = new SSLParameters();
+                @SuppressWarnings("unused")
+                Boolean order = (Boolean) getUseCipherSuitesOrderMethod.invoke(parameters);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "order" local variable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://garage.netty.io/sonarqube/coding_rules#rule_key=squid%3AS1481)